checkstyle/import-control-share.xml

```diff
@@ -45,7 +45,7 @@
       <allow pkg="org.apache.kafka.server.share" />
 
       <subpackage name="fetch">
-        <allow class="org.apache.kafka.storage.internals.log.FetchParams"/>
```

Comment:
Can you check other import control files to see if any others need cleanup?

Reply:
I checked elsewhere but can't find anything to clean.